ALUM/ALUM/Views/SignUpPage/SignUpJoinAs.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  SignUpJoinAs.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/27/23.
+//
+
+import SwiftUI
+
+struct SignUpJoinAs: View {
+
+    enum JoinType {
+        case mentee
+        case mentor
+    }
+    
+    @ObservedObject var viewModel: SignUpPageViewModel
+    @Environment(\.dismiss) var dismiss
+    @State var selectedType: JoinType? = nil
+    
+    var body: some View {
+        ZStack {
+            Color("ALUM White 2").edgesIgnoringSafeArea(.all)
+                .overlay(alignment: .top) {
+                    ProgressBarComponent(nodes: 3, filledNodes: 1, activeNode: 2)
+                        .frame(alignment: .top)
+                        .frame(maxWidth: .infinity)
+                        .padding()
+                        .background(Color.white.ignoresSafeArea(edges: .top))
+                        .frame(maxWidth: .infinity)
+                }
+            ScrollView {
+                VStack {
+                    HStack {
+                        Text("I want to join as a...")
+                            .font(.custom("Metropolis-Regular", size: 34))
+                            .foregroundColor(Color("NeutralGray3"))
+                            .frame(width: 306, height: 41)
+                            .padding(.top, 8)
+                        Spacer()
+                    }
+                    .padding(.bottom, 32)
+
+                    SignUpJoinOption(title: "Mentee",
+                                   description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+                                   isSelected: selectedType == .mentee)
+                        .onTapGesture {
+                            selectedType = .mentee
+                            viewModel.isMentee = true
+                            viewModel.isMentor = false
+                        }
+                        .padding(.bottom, 32)
+                    
+                    SignUpJoinOption(title: "Mentor",
+                                   description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+                                   isSelected: selectedType == .mentor)
+                        .onTapGesture {
+                            selectedType = .mentor
+                            viewModel.isMentee = false
+                            viewModel.isMentor = true
+                        }
+                        .padding(.bottom, 137)
+
+                    HStack {
```

Comment:
action - We should not put this HStack (Back, Continue buttons) inside the ScrollView. User only needs to scroll on the editable stuff so lets put this outside the ScrollView.

Currently, if you notice there is some blank space after the Back/Continue buttons. This is caused by the ScrollView so once the buttons are out of it, that'll be fixed as well.

Reply:
Based on a comment (you will see in few lines), we can adjust the structure of this and other signup screens to be - 
```
ZStack {
    Color("background")...
    VStack {
        ProgressBarComponent(nodes: 3, filledNodes: 1, activeNode: 2)
                    .frame(alignment: .top)
                    ...
       ScrollView() {
            ...
       }
      HStack {
           Button("Back")...
           Button("Continue").... 
     }
    }
}
```